### ISSUE TITLE
Improvement: Refactor code to use dict.get() method

### ISSUE THREAD

**Issue by codesankalp** [anitab-org/mentorship-backend#1065]
## Description
As a developer,
I need code to be consistent and error-free.

## Mocks
https://github.com/anitab-org/mentorship-backend/blob/d52d77706aad4a486f2417419bb4c4c6d2ace3a6/app/api/dao/user.py#L42-L46
For example this can be replaced by the get method i.e. `data.get("KEY", "DEFAULT_VALUE")` to avoid KeyError

### Update [Required]
- [ ] Use get method in all code to avoid the raise of KeyError
- [ ] Handle exceptions for default value

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
2 hours

**Comment by harsh7in** [comment]
hey @codesankalp can i work on this ?

**Comment by codesankalp** [comment]
Sure @harsh-9in :smile:. Assigning you!

**Comment by vj-codes** [comment]
@codesankalp hey was this issue update discussed anywhere?

**Comment by codesankalp** [comment]
@vj-codes This is the referenced comment: https://github.com/anitab-org/stem-diverse-tv/issues/146#issuecomment-808977826

**Comment by isabelcosta** [comment]
@vj-codes thank you for confirming it! I saw the issue being created on stem diverse and thought it could be a little improvement here :)

cc @codesankalp

**Comment by vj-codes** [comment]
@harsh-9in any updates on this?

**Comment by vj-codes** [comment]
@harsh-9in if you have any questions about the issue or setting up the project, you can ask us here in comments or on [zulip](http://anitab-org.zulipchat.com) too :)

**Comment by aakankshaagr** [comment]
@vj-codes Can I work on this issue?

**Comment by aakankshaagr** [comment]
@epicadk I have created a PR please check if any changes are required

**Comment by epicadk** [comment]
@codesankalp I don't get this? Why would we need a default value here? The data that comes from the user is already validated and needs to have these keys.

![image](https://user-images.githubusercontent.com/56596662/119455848-a734ec80-bd57-11eb-9e25-81fa06978673.png)

**Comment by codesankalp** [comment]
> @codesankalp I don't get this? Why would we need a default value here? The data that comes from the user is already validated and needs to have these keys.

Yes, it is but when you see the code there is a lot of repetition of code for validation which can be reduced using `dict.get()`

In the description, I just gave the example of dao/user.py but what I want to convey is to use dict.get method wherever necessary so that code doesn't look repetitive.

Example:
https://github.com/anitab-org/mentorship-backend/blob/25d8d99b3531f34a84343970be781b83f07d85e8/app/api/dao/user.py#L244-L248

This can be written as:
```data.get("bio", None)```

This needs to be changed in the whole code.

**Comment by aakankshaagr** [comment]
@vj-codes @epicadk @codesankalp I have created a PR. Please check, if any changes are required. 
Run tests are failing due to black (formatting issue) and I don't know the command for that. Can someone please tell me the command to remove that issue?

**Comment by aakankshaagr** [comment]
its working fine for invalid email
![image](https://user-images.githubusercontent.com/71868375/119561356-b1400500-bdc2-11eb-9581-cb8570113f46.png)
but when I enter an existing email it's showing an internal server error
![image](https://user-images.githubusercontent.com/71868375/119561554-f6643700-bdc2-11eb-8dc0-fd8fde9b7a5c.png)
I don't understand why this is happening?
Till yesterday it was working fine.
